src/multi_vector_simulator/utils/constants.py

```diff
@@ -242,15 +243,15 @@
 TYPE_NONE = "None"
 TYPE_FLOAT = "float"
 
-EXTRA_CSV_PARAMETERS = {
+KNOWN_EXTRA_PARAMETERS = {
     UNIT: {
         DEFAULT_VALUE: "NA",
         UNIT: TYPE_STR,
         WARNING_TEXT: "defines the unit of power provided by a DSO (Values: str). ",
         REQUIRED_IN_CSV_ELEMENTS: [ENERGY_PROVIDERS],
     },
     MAXIMUM_CAP: {
-        DEFAULT_VALUE: None,
+        DEFAULT_VALUE: 1e17,
```

Comment:
For me, `MAXIMUM_CAP==None` also works. However, you are not changing the `MAXIMUM_EMISSIONS` but the `MAXIMUM_CAP` here! Probably you have weird costs in your input parameters? Which inputs are you using?
```suggestion
        DEFAULT_VALUE: None,
```